article/app/pages/StoryHtmlPage.scala

```diff
@@ -11,19 +11,20 @@ import views.html.fragments._
 import views.html.fragments.commercial.{pageSkin, survey}
 import views.html.fragments.page._
 import views.html.fragments.page.body.{bodyTag, breakingNewsDiv, skipToMainContent, twentyFourSevenTraining}
-import views.html.fragments.page.head.stylesheets.{criticalStyleInline, criticalStyleLink, styles}
+import views.html.fragments.page.head.stylesheets.{atomsCSSInline, criticalStyleInline, criticalStyleLink, styles}
 import views.html.fragments.page.head.{fixIEReferenceErrors, headTag, titleTag}
 
 object StoryHtmlPage {
 
-  def allStyles(implicit applicationContext: ApplicationContext): Styles = new Styles {
+  def allStyles(implicit applicationContext: ApplicationContext, page: Page): Styles = new Styles {
     override def criticalCssLink: Html = criticalStyleLink("content")
     override def criticalCssInline: Html = criticalStyleInline(Html(common.Assets.css.head(None)))
     override def linkCss: Html = stylesheetLink("stylesheets/content.css")
     override def oldIECriticalCss: Html = stylesheetLink("stylesheets/old-ie.head.content.css")
     override def oldIELinkCss: Html = stylesheetLink("stylesheets/old-ie.content.css")
     override def IE9LinkCss: Html = stylesheetLink("stylesheets/ie9.head.content.css")
     override def IE9CriticalCss: Html = stylesheetLink("stylesheets/ie9.content.css")
+    override def atomsCSS: Option[Html] = Some(atomsCSSInline(page))
```

Comment:
Unless you need the atomCss to be in a specific place you can keep things simple by not adding it here but after the `style(allStyles)` in the `html` [method](https://github.com/guardian/frontend/pull/17968/files#diff-814f684a6185ffdf5d7bd0d8e414604eR46):
```
htmlTag(
       headTag(
         titleTag(),
         metaData(),
         head,
         styles(allStyles),
         atomsCSSInline(),
         fixIEReferenceErrors(),
```